scio-core/src/main/scala/com/spotify/scio/values/SideInput.scala

```diff
@@ -42,6 +42,42 @@ trait SideInput[T] extends Serializable {
   private[values] val view: PCollectionView[_]
 }
 
+object SideInput {
```

Comment:
I know it's silly but a one liner doc "Companion object of ..." so it shows up nicely in API site?